src/test-utils.h

```diff
@@ -8,6 +8,11 @@
 #ifndef TEST_UTILS_H
 #define TEST_UTILS_H
 
+// Arduino has two problems with the tests:
+// 1. The test-utils are duplicated w/ other Blockchain Commons packages.
```

Comment:
The test files should *not* be part of the library build. See the makefile.

Reply:
The arduino world doesn't use config or makefiles.  There is no "build" in the sense that you defined one.  They grab everything in `src`.

For the record, I don't think this makes sense either ...